libs/sr-formula/src/data/util/read.ts

```diff
@@ -60,25 +60,25 @@ export class Read extends TypedRead<Tag, Read> {
 
   // Damage Type
   get physical() {
-    return super.with('dt', 'Physical')
+    return super.with('type', 'Physical')
```

Comment:
`type` might be a tad confusing, `dt` is lantua's annoying shorthand, can you call it `dmgType`?